src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/AdmissionState/RoundedLabeledView/RoundedLabeledView.swift

```diff
@@ -57,9 +60,16 @@ class RoundedLabeledView: UIView {
 
 	// MARK: - Internal
 
-	func configure(title: String?, gradientType: GradientView.GradientType) {
+	func configure(title: String?, fontColor: UIColor? = .enaColor(for: .textContrast), image: UIImage? = nil, gradientType: GradientView.GradientType) {
 		titleLabel.text = title
+		titleLabel.textColor = fontColor
 		accessibilityLabel = title
+		
+		if image != nil {
+			imageView.image = image
```

Comment:
Please add `imageView.isHidden = false` for cases when you configure with `image == nil` and then (subscription for e.g.) with `image != nil`.

Reply:
Good catch